### DISCUSSION TITLE
sh: 1: create-docusaurus: Permission denied

### DISCUSSION THREAD

**Discussion by flamecopper** [facebook/docusaurus#8513] [General]
root@osboxes:/home/osboxes/Desktop# npx create-docusaurus@latest my-website classic
sh: 1: create-docusaurus: Permission denied

**Comment by auchynnikau** [comment]
This error message is usually caused when you are trying to execute a file that you don't have permission to run. To fix this issue, you will need to change the permissions of the file so that you have the necessary permission to execute it. You can do this using the `chmod` command.

For example: ```chmod a+x create-docusaurus```;

This will add execute permission for all users.

**Reply by thompcd** [reply]
I'd forgotten I created a new WSL instance. I forgot to setup my permissions. Thanks!
